webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.js

```diff
@@ -59,7 +61,10 @@ const PageLayout = ({
             )}
           </Col>
           <Col id="title_action" md={searchable ? 6 : 8}>
-            <div className="btn-toolbar pull-right">{toolbarButtons}</div>
+            <div className="btn-toolbar pull-right">
+              {toolbarButtons}
+              {exportURL && <ExportButton url={exportURL} />}
```

Comment:
Shouldn't the export button just be one of the toolbarButtons defined by the page? In some pages the location of the button changes - sometimes it isn't the right most one, so it would be better imho to have the page pass it as part of the toolbar buttons array

Reply:
This wasn't addressed, only another button and another parameter added here instead.

Reply:
Foreman should be consistent with button placement imo and these 2 buttons are the only one used in core. New buttons can be added by using Fill with different weights

Reply:
these buttons are not present on all pages, and there is also a "create" button that is common. No plugin adds another button to all pages, so using a fill instead of having the page just pass the toolbarButtons to the layout sounds like complicating this.
imho the specific page should define which buttons to enable, this logic should not be in the page layout. If we want consistent layout let's add a "ToolbarButtons" or "DefaultButtons" component or something that handles this logic.